### PR TITLE
Modifying the Azure Pipeline so that CentOS 7.6 gets additional required commands

### PR DESCRIPTION
**Pull request by ignacionr**
For building on CentOS 7.6, we need to enable the development tools newer version, each time we issue a command.

### CODE REVIEW

File: CI/Linux/linux-build.yml [hunk 1]
```diff
@@ -14,6 +14,19 @@ steps:
...
+      cd build-release
+      yum install -y centos-release-scl-rh
+      yum install -y devtoolset-9-gcc-c++
+      scl enable devtoolset-9 'cmake3 -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=$(pwd)/rpm/BUILDROOT/usr ..;'
+      . ./version
+      scl enable devtoolset-9 'make -j'
+    condition: eq(variables['packager'],'centos76-rpm')
+    displayName: 'Build Release rpm'
+    env:
+      GIT_COMMIT: $(Build.SourceVersion)
```
Comment (SerhiiGadzhilov) [inline CI/Linux/linux-build.yml:28]: As I can see the “devtoolset” was added only for the “Standard” version build. 
Maybe the “Enterprise” build should be updated as well?
Isn't it better to include the "devtoolset" into docker image for CentOS 7.6?

**Review by SerhiiGadzhilov — CHANGES_REQUESTED** [review]

File: CI/Linux/linux-build.yml (hunk 1 above)
Comment (andreacndn) [reply CI/Linux/linux-build.yml:28]: Agreed that the requirement was to update the CentOS docker image

**Review by andreacndn — CHANGES_REQUESTED** [review]
Agreed that the requirement was to update the CentOS docker image

**Comment by SerhiiGadzhilov** [comment]
Fixed under #6937

**Pull request closed without merging**